test/jackdaw/streams_test.clj

```diff
@@ -333,8 +357,8 @@
 
       (produce-a 1 1 1)
       (produce-b 100 2 2)
-      (is (= [1 1] (mock/consume driver topic-c)))
-      (is (= [2 2] (mock/consume driver topic-c)))))
+      (is (= [1 1] (mock/get-keyval driver topic-c)))
```

Comment:
get-keyval was particularly made for here.
We could easily call consume directly and convert to key-val as well